README.md

```diff
@@ -136,6 +136,12 @@ To install just write into your Cartfile:
 github "hyperoslo/Hue"
 ```
 
+To install **Hue** using [Swift Package Manager](https://swift.org/package-manager) with Xcode 11, just follow the instructions at <https://developer.apple.com/documentation/swift_packages> and import the platform specific library to the project:
+
+```swift
+import HueAppKit // or HueUIKit
```

Comment:
@bcylin Is this package/framework name by convention? Just wondering if we could call it `Hue` like we do for CocoaPods etc.

Reply:
@zenangst thanks for the review. It was a workaround to use different source files for macOS and iOS because the [Swift Package description](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md#methods-5
) currently doesn't support platform specific targets like CocoaPods does.

There's an alternative way to wrap the source code inside the available module:

```swift
#if canImport(module)
// ...
#endif
```

I looked into other projects and it seems it's the preferred solution. So I updated it to a single library module. Now it should work using `import Hue`.

Reply:
Thanks a lot buddy, let's get this merged!